server.js

```diff
@@ -10,36 +10,11 @@ var exphbs = require('express-handlebars')
 const path = require('path')
 const mysql = require('mysql')
 
-
-
-// For Passport
- 
-app.use(session({ secret: 'keyboard cat',resave: true, saveUninitialized:true})); // session secret
-
-app.use(passport.initialize());
-
-app.use(passport.session()); // persistent login sessions
 const PORT = process.env.PORT || 5000;
 
-
-app.get('/', function(req, res) {
-
-  res.send('Welcome')
-
-  res.render(path.join(__dirname, "landing.hbs"))
-});
-app.get('/about', function(req, res) {
-  res.render(path.join(__dirname, "about.hbs"))
-});app.get('/contact', function(req, res) {
-  res.render(path.join(__dirname, "contact.hbs"))
-});app.get('/index1.2', function(req, res) {
-  res.render(path.join(__dirname, "profile.hbs"))
-});app.get('/index1.3', function(req, res) {
-  res.render(path.join(__dirname, "signin.hbs"))
-
-});
 // Serving up static assets //
-app.use(express.static('client/build'));
+//app.use(express.static('client/build'));
+app.use(express.static("public"));
```

Comment:
This is needed for express to be able to route static files from the public folder without specific routes for each file.